<launch>
    <arg name="pkg_name" value="ros_example"/>
    <arg name="node_name" default="node_example"/>
    <arg name="veh" default="laptop"/>
    <arg name="local" default="false"/>

    <group ns="$(arg veh)">

        <node unless="$(arg local)"  name="$(arg node_name)" pkg="$(arg pkg_name)"
              type="$(arg node_name).py" output="screen">
        </node>
    </group>
</launch>
